app/_components/nav-drawer/NavDrawer.tsx

```diff
@@ -59,13 +59,13 @@ const NavDrawer = () => {
                 <Image
                   src={'/EDPN_logo_dark_background.png'}
```

Comment:
Considering the dimensions of the header, instead of the EDPN text, really we should have the full project name. The spelled version of the logo is ideal for this, and I have made a version without the background for this purpose.
![EDPN_logo_spelled_transparent](https://github.com/ed-pilots-network/frontend/assets/50422789/911b6998-fe8a-4914-ba30-9f10c9a135a3)

Reply:
![image](https://github.com/ed-pilots-network/frontend/assets/62995185/df5c2238-a46a-4d2b-b037-d6dd947d9359)